File: Selenium/Activities/Activity5.py
from selenium import webdriver
from selenium.webdriver.common.by import By
with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/dynamic-controls")
    print(f"the title of main page is: {driver.title}")

    checkBox = driver.find_element(By.ID,"checkbox")
    checkBox.is_displayed()

    toggleButton = driver.find_element(By.XPATH,"//button[text()='Toggle Checkbox']")
    toggleButton.click()

    checkBox.is_displayed()
    print(f"The checkbox is visble: {checkBox.is_displayed()}")

    driver.quit()

File: Selenium/Activities/Activity5.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity5 {
	public static void main(String args[]) {
		WebDriver driver = new FirefoxDriver();
		driver.get("https://training-support.net/webelements/dynamic-controls");
		System.out.println("The title of the main page is: "+driver.getTitle());
		
		WebElement checkbox = driver.findElement(By.id("checkbox"));
		checkbox.isDisplayed();
		
		WebElement toggleButton = driver.findElement(By.xpath("//button[text()='Toggle Checkbox']"));
		toggleButton.click();
		
		
		checkbox.isDisplayed();
	
		
		System.out.println("The checkbox is visible: "+checkbox.isDisplayed());
		
		driver.quit();
		
		
		
		
		
	
	}

}
